CONTRIBUTING.md

```diff
@@ -62,6 +62,15 @@ $ git checkout new-feature
 Before commiting any code, please read our
 [Contributor Agreement](http://github.com/SciRuby/sciruby/wiki/Contributor-Agreement).
 
+## C/C++ style guide
```

Comment:
Though, now that I think of it, we have an (incredibly incomplete) style guide section in the wiki that might be easier to use than requiring a pull request every time: https://github.com/SciRuby/nmatrix/wiki/NMatrix-Developer-Guide#code-style